charts/minikube.md

```diff
@@ -60,30 +63,66 @@ helm install --name zenko \
 
 To view the Kubernetes dashboard type the following and will launch the dashboard in your default browser:
 ```shell
-minikube dashboard
+$ minikube dashboard
 ```
 #### NOTE that once you helm install, it may take several minutes for all the pods to load and stabilize
 
 The endpoint can now be accessed via the kubernetes cluster ip (run ```minikube ip``` to display the cluster ip)
 
+## Testing your deployment
+
+To test your minikube deployment, please refer to our client configuration guide,
```

Comment:
Since Orbit is enabled by default, it might be a good idea to have a link to `docs/orbit_registration.md` here.

Reply:
Added a link here, and rephrased the second one at the end of the "testing" Paragraph